.github/workflows/ci.yml

```diff
@@ -76,35 +57,22 @@ jobs:
   bitrot:
     name: Bitrot check
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: 1.60.0
-          override: true
       # Build benchmarks to prevent bitrot
       - name: Build benchmarks
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: --all --benches
+        run: cargo build --all --benches
 
   clippy:
-    name: Clippy (1.60.0)
+    name: Clippy (MSRV)
     timeout-minutes: 30
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: 1.60.0
-          components: clippy
-          override: true
       - name: Run clippy
         uses: actions-rs/clippy-check@v1
```

Comment:
I did some searching and found a few possible alternatives for this, but I'm not currently sold on any of them:
- https://crates.io/crates/clippy-sarif
  - Integrates with GitHub Advanced Security, so the lints appear as security warnings.
- https://github.com/giraffate/clippy-action
  - Requires integration with `reviewdog` AFAICT.
- https://github.com/qernal/github-actions-rust-clippy
  - Looks equivalent, but also appears to only be used by its author.